app/controllers/publish/users_controller.rb

```diff
@@ -3,7 +3,7 @@ class UsersController < PublishController
     before_action :authorize_provider
 
     def index
-      @users = provider.users
+      @users = provider.users.order("LOWER(first_name), LOWER(last_name)")
```

Comment:
I think this should be moved down into the model under a scope then it'll read a little better in the controller:

`provider.users.in_name_order` or something

Reply:
👍🏼 https://github.com/DFE-Digital/publish-teacher-training/pull/3235/commits/3478c10439b503e84e50944d923dec7321b8be0f